api-catalog-ui/frontend/src/components/Wizard/configs/wizard_regex_restrictions.jsx

```diff
@@ -2,6 +2,10 @@
 export const wizRegex = {
     gatewayUrl: { value: '^(/[a-z]+\\/v\\d+)$', tooltip: 'Format: /api/vX, Example: /api/v1' },
     version: { value: '^(\\d+)\\.(\\d+)\\.(\\d+)$', tooltip: 'Semantic versioning expected, example: 1.0.7' },
+    ipAddress: {
+        value: '^((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\\.){3}(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$',
+        tooltip: 'IP Address v4 expected, example: 127.0.0.1',
```

Comment:
We will need to reflect the change with the IPv6 support claim.

Reply:
Yes, good point.